RELOAD-MODULES: changes to LIS drivers handling

chvalean opened the pull request:
1. add check that hv_utils cannot be unloaded.
modprobe returns that the module is in use and cannot be unloaded.
2. remove hid_hyperv reload from loop, we care about the netvsc driver. reloading multiple lis drivers was the original design, as other modules might have been unloaded. nowadays, that is no longer the case.

---

Sample output after changes, previously it was a mix of cannot unload hv_utils and more text for reloading hid_hyperv.

```
printf '%s\n' hv_netvsc hv_utils hv_balloon hid_hyperv hyperv_keyboard hyperv_fb
grep -xq hv_netvsc
modprobe -r hv_netvsc
sleep 1
modprobe hv_netvsc
sleep 1
pass=7
```

chvalean commented on Testscripts/Linux/STOR-Verify-Sector-Size.sh:
```diff
@@ -0,0 +1,60 @@
+    parted $drive_name mklabel msdos
+    parted $drive_name mkpart primary 0% 100%
+else
+    LogErr "Unable to create partition"
```
parted not installed would be the accurate message. wondering if it's worth adding the code to install the package

asofron replied:
Update to  "parted not installed"

chvalean committed:
Update RELOAD-MODULES.sh

1. add check that hv_utils cannot be unloaded, modprobe will return that the module is in use and cannot be unloaded.
2. remove hid_hyperv reload from loop, we care about the netvsc driver. reloading multiple lis drivers was the original design, as other modules might have been unloaded. nowadays, that is no longer the case.

chvalean committed:
Update RELOAD-MODULES.sh

chvalean committed:
Update RELOAD-MODULES.sh
